15.1 docs: `unstablePersistentCaching` (#73256)
vercel/next.js: docs/01-app/03-api-reference/05-config/01-next-config-js/turbo.mdx

The following options are available for the `turbo` configuration:

| Option                    | Type                                            | Description                                                                |
| ------------------------- | ----------------------------------------------- | -------------------------------------------------------------------------- |

> | `loaders`                 | `Record<string, TurboLoaderItem[]>`             | List of webpack loaders to apply when running with Turbopack.              |

Reviewer: That doesn't exist anymore. It has been removed in favor of `rules`

Author: Removed 👍🏼

> | `memoryLimit`             | `number`                                        | A target memory limit for turbo, in bytes.                                 |

Reviewer: This shouldn't be documented for now...

Author: Removed 👍🏼

| `moduleIdStrategy`        | `'named'` or `'deterministic'`                  | Assign module IDs                                                          |
| `minify`                  | `boolean`                                       | Enable minification. Defaults to true in build mode and false in dev mode. |
| `resolveAlias`            | `Record<string>`                                | Map aliased imports to modules to load in their place.                     |
| `resolveExtensions`       | `string[]`                                      | List of extensions to resolve when importing files.                        |
| `rules`                   | `Record<string, TurboRuleConfigItemOrShortcut>` | List of unsupported webpack loaders to apply when running with Turbopack.  |
| `root`                    | `string`                                        | The repo root directory.                                                   |
| `treeShaking`             | `boolean`                                       | Enable tree shaking for the turbopack dev server and build.                |

> | `unstablePersistentCache` | `boolean`                                       | Enable the persistent cache for turbopack dev and build servers.           |

Reviewer: ```suggestion
| `unstablePersistentCaching` | `boolean`                                       | Enable the persistent cache for Turbopack dev server and build.           |
```

## Examples

### Enabling Persistent Caching

Turbopack's Persistent Caching allows compilation work to persist across restarts of the development server or builds, reducing cold start times for all applications by caching previously compiled work, and only recompiling what has changed.

> To enable persistent caching, set the `unstablePersistentCache` option to `true` in `next.config.js`:

Reviewer: ```suggestion
To enable persistent caching, set the `unstablePersistentCaching` option to `true` in `next.config.js`:
```

```ts filename="next.config.ts" switcher
import type { NextConfig } from 'next'

const nextConfig: NextConfig = {
  experimental: {
    turbo: {

>       unstablePersistentCache: true,

Reviewer: ```suggestion
      unstablePersistentCaching: true,
```

    },
  },
}

export default nextConfig
```

```js filename="next.config.js" switcher
module.exports = {
  experimental: {
    turbo: {

>       unstablePersistentCache: true,

Reviewer: ```suggestion
      unstablePersistentCaching: true,
```
